using-timescaledb/distributed-hypertables.md

```diff
@@ -1,41 +1,47 @@
-multi-node TimescaleDB architecture
-([architecture][]). All the nodes are TimescaleDB instances,
-i.e., hosts with a running PostgreSQL server and loaded TimescaleDB extension.
-While the data nodes store distributed chunks, the access node is
-the entry point for clients to access distributed hypertables.
-
-For more information about setting up a multi-node architecture, including
-configuring access nodes and data nodes, please follow our [getting started
-with multi-node instructions][getting-started-multi-node].
-
->:WARNING: There are some [limitations][distributed-hypertable-limitations] to
-using distributed hypertables that might be good to review before getting
-started.
-
-### Creating Distributed Hypertables [](create)
-
-As with normal hypertables, start by creating a regular
-table to hold the data, then transform it into a distributed
-hypertable. In contrast to creating a normal hypertable, we use
-the function
-[`create_distributed_hypertable`][create_distributed_hypertable].
+Distributed hypertables extend regular hypertables with the ability to
+store data chunks across multiple data nodes for better scale-out
+performance. Prior to creating a distributed hypertable, however,
+TimescaleDB must be [set up for multi-node
+usage][getting-started-multi-node].
+
+In most cases, using a distributed hypertable is similar to using a
+regular hypertable, including inserting, querying, and altering
+it. Thus, for basic usage, please review the [documentation for regular
+hypertables][using-hypertables].
+
+For inserts and queries, however, a distributed hypertable has unique
+performance characteristics and there are also additional limitations
+due to its distributed nature. For instance, query performance is
+heavily dependent on the ability to *push down* work to data nodes,
+which in turn ties into how data is partitioned across the nodes. If
+push down is not possible, or the query does not involve many data
+nodes, the query performance of a distributed hypertable will likely
+be worse than that of a regular hypertable due to the additional
+network overhead.
+
+Note, also, that distributed hypertables can live alongside
+non-distributed tables and other objects; in fact, no objects are
+distributed by default. Interactions between distributed hypertables
+and non-distributed objects might not have the expected behavior. For
+instance, setting permissions on a distributed hypertable only works
+if the roles involved exist identically on all data nodes. Further,
+joins between a local table and a distributed hypertable requires
+fetching the raw data from data nodes and performing the join locally.
+
+>:WARNING: Distributed hypertables are not yet recommended for
+production use. For more information, please review the current
+[limitations][distributed-hypertable-limitations]. You can also
+[contact us][contact] or join the #multinode channel in our [community
+Slack][slack].
+
+### Creating a Distributed Hypertable [](create)
```

Comment:
Should this be ## or ###, @solugebefola ?  I think these should all just be one level below the top, so ### -> ##, and #### -> ###?  (But I remember with API there was something funny, so letting solar opine).